desktop/fastlane/.gitignore

```diff
@@ -0,0 +1,2 @@
+report.xml
+README.md
```

Comment:
```suggestion
report.xml
README.md
.env
```

Reply:
@iangmaia here's my mental model of the setup:

- We want to track `fastlane/example.env` because it's the template devs should use to set up the local environment if needed
- We don't plan on having additional `.env` files and currently do not support using them
- We also don't have a `.env` folder or file in Fastlane at the moment to ignore with this pattern

Reply:
I imagined this would still be a possibility; given `fastlane/example.env` I thought for a moment we'd use a local `.env`, but I think this is key:
> We don't plan on having additional .env files and currently do not support using them

Reply:
To be fair, `.env` files are a common pattern in Ruby and other ecosystems, so much so that [the `dotenv` gem provides an auto-load mechanism for any `.env` file present in the current directory](https://github.com/bkeepers/dotenv), and that is at least used by _fastlane_.

Which means that anyone can put their own custom secrets in a `fastlane/.env` file and fastlane will auto-load it.
In fact, we've considered in the past removing our custom logic that we've added in the `Fastfile` of many of our repos trying to load a custom `USER_ENV_FILE`, in favor of this common community convention to rely on the existing automatic mechanism that _fastlane_ and `dotenv` already support regarding auto-loading an `.env` file in the current directory if one exists.

While that's true that currently most of our repos instead have this custom logic to load our custom `USER_ENV_FILE` and check its presence etc:
 - One could wonder if switching to the more default convention of using a local `.env` file would be worth it to simplify our `Fastfile` and be more in line with Ruby community conventions. That being said, I'm not 100% sure that'd be the right move for us, as one of the reasons we moved those env files in `$HOME/.a8c-apps` and especially outside of the repo itself was to avoid the risk of accidentally committing any secrets to the repo
 - But still, if anyone wanted to use a `fastlane/.env` file to store their secrets, that would still technically work out of the box. And if we want to make sure people who would be potentially tempted to do that still don't risk accidentally commit said secrets to git, adding `.env` to the `.gitignore` feels like a good security measure, and wouldn't hurt anyway?

Reply:
I see...

> But still, if anyone wanted to use a fastlane/.env file to store their secrets, that would still technically work out of the box. And if we want to make sure people who would be potentially tempted to do that still don't risk accidentally commit said secrets to git, adding .env to the .gitignore feels like a good security measure, and wouldn't hurt anyway?

:thinking_face: I guess that would  work in the sense that if one adds `fastlane/.env` and leaves `~/.a8c-apps/<env for the project>` empty, then our logic to pick up the env values would not complain because Fastlane would have loaded `fastlane/.env` and provided the values.

But is that something we want to encourage, by implicitly supporting it? 

At the same time, the cost of not ignoring the `.env` file would be risking the file from leaking into the repo 🫤 

Better safe than sorry I guess, so I guess I'm convinced on adding `.env` to the `.gitignore` 👍 

E.g.

```
# The recommended way to define environment values is to copy example.env to ~/.a8c-apps/wordpress-desktop.env.
# But given Fastlane automatically loads .env if present, let's ignore it in case people want to use that approach instead.
.env
```

Reply:
Yep more of a safety net than anything.
I like the comment you added to provide context 👍